### DISCUSSION TITLE
Why I can't use it?

### DISCUSSION THREAD

**Discussion by ffvv0123** [community/community#42109] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://user-images.githubusercontent.com/63843822/208385298-68ee76d9-874c-4ba8-adca-de86d51e4489.png)
![image](https://user-images.githubusercontent.com/63843822/208385344-a09e38f2-8177-4d49-82d7-a952b0ed4a7c.png)

I used free-trial copilot last 60 days. 
Now I accessed student pack.
However, it appears like the picture above, and when I press that button, it connects to the next site.
https://github.com/settings/billing
![image](https://user-images.githubusercontent.com/63843822/208385761-72eff551-cba1-4830-b668-cf811c15a444.png)

After being moved to the site, I can't take any action.
How can I use copilot?

**Comment by iamcjwilliams** [comment]
@ffvv0123 Sorry for the trouble here, please visit [here](https://education.github.com/discount_requests/pack_application) in order to verify your student access.
